lightning-graph-sync/src/lib.rs

```diff
@@ -0,0 +1,138 @@
+#![deny(missing_docs)]
+#![deny(unsafe_code)]
+#![deny(broken_intra_doc_links)]
+#![deny(non_upper_case_globals)]
+#![deny(non_camel_case_types)]
+#![deny(non_snake_case)]
+#![deny(unused_mut)]
+#![deny(unused_variables)]
+#![deny(unused_imports)]
+//! This crate exposes functionality to rapidly sync gossip data, aimed primarily at mobile
+//! devices.
+
+// Allow and import test features for benching
+#![cfg_attr(all(any(test, feature = "_test_utils"), feature = "_bench_unstable"), feature(test))]
```

Comment:
if we drop _test_utils, this could be simplified to 
`#![cfg_attr(all(test, feature = "_bench_unstable"), feature(test))]`